config.rb

```diff
@@ -40,7 +40,7 @@
 
 # Build Configuration
 configure :build do
-  activate :asset_hash
+  activate :asset_hash, ignore: /remotelock-config.js/
```

Comment:
I don't quite remember why this is necessary

Reply:
This is necessary to prevent it from including a hash in this filename, so we could replace the file on deploy.

Thinking on this now, maybe we could copy the hashed filename before replacing it. 🤔

Reply:
I think the current solution is better, honestly

Reply:
I think the current solution is better, honestly